src/frontend/src/icp/derived/icrc.derived.ts

```diff
@@ -130,3 +131,8 @@ export const enabledIcrcTokens: Readable<IcToken[]> = derived(
 		...$enabledIcrcCustomTokens
 	]
 );
+
+export const enabledIcrcTokensLedgerCanisterIds: Readable<LedgerCanisterIdText[]> = derived(
```

Comment:
`enabledIcrcLedgerCanisterIds` to shorten it a bit?

Reply:
done